runtime/common/src/zenlink.rs

```diff
@@ -98,6 +98,7 @@ pub fn currency_id_to_zenlink_id(
 				((token2_type as u64) << 40);
 			Ok(ZenlinkAssetId { chain_id: parachain_id, asset_type: LOCAL, asset_index: index })
 		},
+		_ => Err(()),
```

Comment:
Nit: We could also make it explicit here by creating an extra match branch for the `Token()` and returning an error. This way, we kind of show that we deliberately decided not to support it.

Reply:
I see, also maintaining the `_=>` branch for extra cases. Seems good.